crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -917,7 +970,7 @@ async fn send_archived_segment_notification(
     archived_segment_notification_sender.notify(move || archived_segment_notification);
 
     while acknowledgement_receiver.next().await.is_some() {
-        debug!(
+        info!(
```

Comment:
This will be very annoying for farmers with hundreds of farms connected to the node. Lets change back to debug.